packages/techdocs-common/src/stages/generate/helpers.ts

```diff
@@ -173,9 +173,13 @@ export const validateMkdocsYaml = async (mkdocsYmlPath: string) => {
   const mkdocsYml: any = yaml.load(mkdocsYmlFileString, {
     schema: MKDOCS_SCHEMA,
   });
-  if (mkdocsYml.docs_dir && isAbsolute(normalize(mkdocsYml.docs_dir))) {
+  if (
+    mkdocsYml.docs_dir &&
+    (isAbsolute(normalize(mkdocsYml.docs_dir)) ||
+      normalize(mkdocsYml.docs_dir).match(/^\.\.\//))
+  ) {
     throw new Error(
-      "docs_dir configuration value in mkdocs can't be an absolute directory path for security reasons. Use relative paths instead which are resolved relative to your mkdocs.yml file location.",
+      "docs_dir configuration value in mkdocs can't be an absolute directory or start with ../ for security reasons. Use relative paths instead which are resolved relative to your mkdocs.yml file location.",
     );
```

Comment:
The scaffolder uses the following code to handle this issue:
https://github.com/backstage/backstage/blob/e0bfd3d44888e89cae10bd423f0742f48be8dfc6/plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts#L140-L146

Could this be an option to use it here? I would assume that this would also work on Windows.

Reply:
:+1: Cool, I swapped the implementation to match that